build_tag.sh

```diff
@@ -5,6 +5,11 @@ GIT_ORG=$(echo $GIT_URL|cut -d":" -f2|cut -d"/" -f1)
 GIT_REPO=$(echo $GIT_URL|cut -d"/" -f2|cut -d"." -f1)
 ZIP_NAME=CognitoPreSignupTrigger.zip
 
+if [[ -z "$TAG" ]]; then
```

Comment:
If run as result of a PR merge, then tag will not be supplied?

Reply:
Push to main: `GIT_BRANCH=origin/main`
Tag push: `GIT_BRANCH=refs/tags/{tagname}` (I think)

So the cut command would result in an empty `TAG`